### DISCUSSION TITLE
Golangci-lint does not lint for all files inside subdirectories

### DISCUSSION THREAD

**Discussion by ViralDoshi1906** [golangci/golangci-lint#4877] [Q&A] [answered]
When I run `golangci-lint run -c ./.golangci.yml ` I do not see any linter errors. But when I run it for a specific file like: `golangci-lint run dir1/file1.go -c ./.golangci.yml` I see the Linting issues.I also tried to run it on all files using `./...` but still I don't see any linting errors.
Can someone help me with a possible fix ? Also a strange thing, When I copy my exact golangci.yml file to a separate project it seems to work pretty fine. Please help.

Go version: 1.22.3
Golangci-lint version: 1.59.0

**Comment by ldez** [comment, marked as answer]
#4880
